qemu/tests/block_copy.py

```diff
@@ -321,3 +321,48 @@ def clean(self):
         while self.trash_files:
             tmp_file = self.trash_files.pop()
             utils.system("rm -f %s" % tmp_file, ignore_status=True)
+
+    def wait_for_complete(self):
+        """
+        Check block job status, utils timeout; if still not complete,
+        raise TestFail exception.
+        """
+        params = self.parser_test_args()
+        timeout = params.get("wait_timeout")
+        if self.vm.monitor.protocol == "qmp":
+            self.vm.monitor.clear_event("BLOCK_JOB_COMPLETED")
+        complete = utils_misc.wait_for(lambda: self.is_complete(),
+                                       first=3.0, step=3.0, timeout=timeout)
+        if not complete:
+            raise error.TestFail("Wait block job ready "
+                                 "timeout in %ss" % timeout)
+
+    def is_complete(self):
+        """
+        Check block job completed or not.
+        :return block job status.
+        """
+        return self.vm.monitor.get_event("BLOCK_JOB_COMPLETED")
+
+    def create_file(self):
+        """
+        Create files and record m5 values of them.
+        """
+        params = self.params
+        session = self.get_session()
+        file_names = params["file_names"]
```

Comment:
file_names is a list like "test1 test2", I have tested, "md5sum test1 test2" will sum both files md5 to record.md5, and "md5sum -c record.md5" will output:
test1: OK
test2: OK